dataRetriever/requestHandlers/requesters/equivalentProofsRequester.go

```diff
@@ -0,0 +1,33 @@
+package requesters
+
+// ArgEquivalentProofsRequester is the argument structure used to create a new equivalent proofs requester instance
+type ArgEquivalentProofsRequester struct {
+	ArgBaseRequester
+}
+
+type equivalentProofsRequester struct {
+	*baseRequester
+}
+
+// NewEquivalentProofsRequester returns a new instance of equivalent proofs requester
+func NewEquivalentProofsRequester(args ArgEquivalentProofsRequester) (*equivalentProofsRequester, error) {
+	err := checkArgBase(args.ArgBaseRequester)
+	if err != nil {
+		return nil, err
+	}
+
+	return &equivalentProofsRequester{
+		baseRequester: createBaseRequester(args.ArgBaseRequester),
+	}, nil
+}
+
+// RequestDataFromHashArray requests equivalent proofs data from other peers by having multiple header hashes and the epoch as input
```

Comment:
do we not have also requests for nonce?
I think when syncing we request headers in advance by nonce rather than by hash
So the same should be done for proofs

Reply:
implemented by nonce, will be integrated in the next PR